app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt

```diff
@@ -86,14 +88,35 @@ class   SendReviewFragment : Fragment() {
         }
     }
 
+    fun createSendResultIntent(sendViewModel: SendViewModel, sendResult: String?): Intent {
+        val intent = Intent()
+        intent.putExtra("assetName", sendViewModel.getSelectedAsset().value!!.name)
+        intent.putExtra("amount", sendViewModel.getSelectedSendAmount().toDouble())
+        intent.putExtra("address", sendViewModel.getSelectedAddress().value!!)
+
+        if (sendViewModel.txID != null){
+            intent.putExtra("transactionId", sendViewModel.txID)
+        }
+
+        if (sendResult != null){
```

Comment:
Here and in 97 line there are not enough space before brace
Just to follow the code style in file

Reply:
Thanks, I added spaces